web/app/forms.py

```diff
@@ -14,3 +14,23 @@ class SearchPlayerForm(Form):
     player_name = StringField('Username', validators=[])
     aga_id = StringField('AGA id', validators=[Optional()])
     submit = SubmitField()
+
+class AddTournamentForm(Form):
```

Comment:
FWIW - you can reuse the same TournamentForm for both adding and editing a tournament. Ultimately, the Flask Form's only purpose is to present a form, accept data, and validate it as specified - what you do with the values afterwards (create a new tournament or update an existing one) is up to you.